contribs/gnodev/pkg/events/events.go

```diff
@@ -0,0 +1,91 @@
+package events
+
+import (
+	"fmt"
+
+	"github.com/gnolang/gno/tm2/pkg/amino"
+	abci "github.com/gnolang/gno/tm2/pkg/bft/abci/types"
+	"github.com/gnolang/gno/tm2/pkg/bft/types"
+	"github.com/gnolang/gno/tm2/pkg/std"
+)
+
+type EventType string
+
+const (
+	EvtReload         EventType = "EVENT_NODE_RELOAD"
+	EvtReset          EventType = "EVENT_NODE_RESET"
+	EvtPackagesUpdate EventType = "EVENT_PACKAGES_UPDATE"
+	EvtTxResult       EventType = "EVENT_TX_RESULT"
```

Comment:
After looking at this for a second, I'm OK with having this prefixed with Evt (even if it is a bit redundant)

but I would change the constant values to remove `EVENT_` (this will come in json after "type":, in an event system, so it's quite clear what it is already.)